python/doc_generator.py

```diff
@@ -47,7 +54,12 @@ def _get_transform_markdown(transform_yaml: Dict,
         md.h2('Parameters'),
         md.table(transform_yaml['arguments']),
         md.h2('Example'),
+        md.text(DATASET_PREVIEW_TEXT),
```

Comment:
I feel like an header (`H2` `H3` or `H4`)  may be more appropriate for sectioning the documentation over just a text